invenio_records_rest/views.py

```diff
@@ -30,9 +30,11 @@
 import uuid
 from functools import partial, wraps
 
+from elasticsearch.exceptions import RequestError
 from flask import Blueprint, abort, current_app, jsonify, make_response, \
     request, url_for
 from flask.views import MethodView
+from flask_babelex import gettext
```

Comment:
- [ ] please include `Flask-BabelEx` in `setup.py`
- [ ] add configuration for `Babel` to `setup.cfg`
- [ ] run `python setup.py extract_messages` and initialize catalog for english
- [ ] add `.tx` config

(you can generate most of it using cookiecutter-invenio-module)